data/categories.txt

```diff
@@ -8895,7 +9158,4 @@ zh-Hant:^山屋
 sponsored-thor
 en:Collect all the items and win prizes!
 ru:Соберите все и выиграйте призы!
-
-sponsored-halloween
-en:Let's check your luck. Trick or treat?
-ru:Проверим вашу удачу? Нажмите на «Trick» или «Treat»
+ -sponsored-halloween 
```

Comment:
Please, do not change the existing lines in this PR.
It currently shows "+266/-6", it should show "+265/-0"